python/lsst/pipe/tasks/processBrightStars.py

```diff
@@ -353,45 +353,49 @@ def warpStamps(self, stamps, pixCenters):
 
         Parameters
         ----------
-        stamps : `collections.abc.Sequence`
-                     [`afwImage.exposure.exposure.ExposureF`]
+        stamps : `Sequence` [`lsst.afw.image.exposure.ExposureF`]
             Image cutouts centered on a single object.
-        pixCenters : `collections.abc.Sequence` [`geom.Point2D`]
-            Positions of each object's center (as obtained from the refCat),
-            in pixels.
+        pixCenters : `Sequence` [`lsst.geom.Point2D`]
+            Positions of each object's center (from the refCat) in pixels.
 
         Returns
         -------
         result : `lsst.pipe.base.Struct`
             Results as a struct with attributes:
 
             ``warpedStars``
-                List of stamps of warped stars (`list` of `afwImage.maskedImage.maskedImage.MaskedImage`).
+                Stamps of warped stars.
+                    (`list` [`~lsst.pipe.base.PipelineTask`])
```

Comment:
Is this really a list of `PipelineTask`?

Reply:
Thank you for catching this!

Reply:
PS `afwImage.maskedImage.maskedImage.MaskedImage` was never going to work either and needs to be `lsst.afw.image.MaskedImage`.